wordpress-mobile/WordPress-iOS
# [Domain Management] Update my domains row UI

# Testing Steps
- Install & Login Jetpack
- Enable Remote Feature Toggle `Domain Management`
- Go to "Me" screen
- Validate the UI for My Domains row.
- Verify the tap action now results in Push rather than Present.
- Log out & verify the row is not visible
To test:

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
